content/patterns/alert/alert-pattern.html

```diff
@@ -14,42 +14,35 @@
   </head>
   <body>
     <main>
-      <h1>Alert Pattern</h1>
+      <h1>アラート パターン</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
+        <h2>このパターンについて</h2>
         <p>
-          An <a class="role-reference" href="#alert">alert</a> is an element that displays a brief, important message in a way that attracts the user's attention without interrupting the user's task.
-          Dynamically rendered alerts are automatically announced by most screen readers, and in some operating systems, they may trigger an alert sound.
-          It is important to note that, at this time, screen readers do not inform users of alerts that are present on the page before page load completes.
+          <a class="role-reference" href="#alert">アラート</a>は、利用者のタスクを中断することなく利用者の注意を引く方法によって、簡潔で重要なメッセージを表示する要素です。動的にレンダリングされたアラートは、ほとんどのスクリーンリーダーで自動的にアナウンスされ、一部のオペレーティングシステムでは、アラート音が鳴る場合があります。現時点でスクリーンリーダーは、ページの読み込みが完了する前にページ上に存在するアラートを、利用者に通知しないことに注意することが重要です。
         </p>
         <p>
-          Because alerts are intended to provide important and potentially time-sensitive information without interfering with the user's ability to continue working, it is crucial they do not affect keyboard focus.
-          The <a href="../alertdialog/alertdialog-pattern.html">Alert Dialog Pattern</a> is designed for situations where interrupting work flow is necessary.
+          アラートは、利用者の作業を続行する能力を妨げることなく、重要かつ時間的にセンシティブな恐れのある情報を提供することを目的としているため、キーボードフォーカスに影響を与えないようにすることが非常に重要です。<a href="../alertdialog/alertdialog-pattern.html">アラートダイアログ パターン</a>は、ワークフローを中断する必要がある状況のために設計されたものです。
         </p>
         <p>
-          It is also important to avoid designing alerts that disappear automatically.
-          An alert that disappears too quickly can lead to failure to meet
-          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-no-exceptions.html">WCAG 2.0 success criterion 2.2.3</a>.
-          Another critical design consideration is the frequency of interruption caused by alerts.
-          Frequent interruptions inhibit usability for people with visual and cognitive disabilities, which makes meeting the requirements of
-          <a href="https://www.w3.org/TR/UNDERSTANDING-WCAG20/time-limits-postponed.html">WCAG 2.0 success criterion 2.2.4</a> more difficult.
+          アラートが自動的に消えるように設計しないことも重要です。あまりにも早く消えるアラートは、<a href="https://waic.jp/translations/WCAG21/Understanding/no-timing.html">WCAG 2.1 達成基準 2.2.3</a> を満たすことができなくなる可能性があります。アラートによる中断の頻度も重要な設計上の考慮事項です。頻繁な中断は、視覚や認知に障害を持つ人々の利用性を阻害するため、<a href="https://waic.jp/translations/WCAG21/Understanding/interruptions.html">WCAG 2.1 達成基準 2.2.4</a> の要件を満たすことが難しくなります。
```

Comment:
達成基準2.2.4ではinterruptionを「割り込み」と訳しています (2010年版では「中断」だったのですが2016年に変更しました)。ここの interrupting / interruption も「割り込み」としたほうが良いと思います。

```suggestion
          アラートが自動的に消えるように設計しないことも重要です。あまりにも早く消えるアラートは、<a href="https://waic.jp/translations/WCAG21/Understanding/no-timing.html">WCAG 2.1 達成基準 2.2.3</a> を満たすことができなくなる可能性があります。アラートによる割り込みの頻度も重要な設計上の考慮事項です。頻繁な割り込みは、視覚や認知に障害を持つ人々の利用性を阻害するため、<a href="https://waic.jp/translations/WCAG21/Understanding/interruptions.html">WCAG 2.1 達成基準 2.2.4</a> の要件を満たすことが難しくなります。
```